### ISSUE TITLE
Missing example code for accessibility best practices case study

### ISSUE THREAD

**Issue by sebastian-nunez** [angular/angular#55583]
### Describe the problem that you experienced

In the accessibility best practices, the [Case study: Building a custom progress bar](https://angular.dev/best-practices/a11y#case-study-building-a-custom-progress-bar) does not have any example code.

There is an error in the console which may be associated with the preview, I pasted it below.

### Enter the URL of the topic with the problem

https://angular.dev/best-practices/a11y#case-study-building-a-custom-progress-bar

### Describe what you were looking for in the documentation

The code associate with case study.

### Describe the actions that led you to experience the problem

Opened the webpage.

### Describe what you want to experience that would fix the problem

I should be able to see the code snippet associated with the case study.

### Add a screenshot if that helps illustrate the problem

<img width="1431" alt="Screenshot 2024-04-29 at 3 47 11 PM" src="https://github.com/angular/angular/assets/45319275/6dcbfc18-d1fb-47b1-84a3-c94909836b89">

### If this problem caused an exception or error, please paste it here

```true
TypeError: this.previewsComponents[e] is not a function
    at t.loadPreview (main-FOR4QIEM.js:7:521)
    at r.<anonymous> (chunk-DTDHOMFN.js:6:83987)
    at Generator.next (<anonymous>)
    at chunk-J65QSAYV.js:1:1258
    at new c (polyfills-XXUF43KU.js:2:2235)
    at Mw (chunk-J65QSAYV.js:1:1078)
    at r.renderExample (chunk-DTDHOMFN.js:6:83838)
    at r.<anonymous> (chunk-DTDHOMFN.js:6:97150)
    at Generator.next (<anonymous>)
    at chunk-J65QSAYV.js:1:1258
```
```


### If the problem is browser-specific, please specify the device, OS, browser, and version

_No response_

### Provide any additional information here in as much as detail as you can

_No response_

**Comment by sebastian-nunez** [comment]
Same issue and error appear in [Example: Transforming a value exponentially](https://angular.dev/guide/pipes/transform-data#example-transforming-a-value-exponentially).

There code is never rendered.

<img width="1429" alt="image" src="https://github.com/angular/angular/assets/45319275/ff38a1fe-5f45-4730-8bf5-2545cca97c3c">
